types/jest/index.d.ts

```diff
@@ -286,6 +286,11 @@ declare namespace jest {
      * This only works with jest-circus!
      */
     function retryTimes(numRetries: number, options?: { logErrorsBeforeRetry?: boolean }): typeof jest;
+    /**
+     * Replace object[propertyKey] with a value.
+     * The property must already exist on the object. The same property might be replaced multiple times. Returns a Jest replaced property.
+     */
+    function replaceProperty<T, K extends keyof T>(obj: T, key: K, value: T[K]): ReplaceProperty<T[K]>;
```

Comment:
Could you copy JSDoc for `replaceProperty` and `resetAllMocks` from the same commit: https://github.com/DefinitelyTyped/DefinitelyTyped/commit/75bbfbd011749c7d877b09c30951aa54fbad06a0

I took them from Jest repo.

Reply:
Done

Reply:
Also the comment above `resetAllMocks`, please. It must mention `replaceProperty`.